src/main/webapp/app/exercises/programming/shared/code-editor/model/code-editor.model.ts

```diff
@@ -103,3 +103,20 @@ export enum GitConflictState {
     CHECKOUT_CONFLICT = 'CHECKOUT_CONFLICT',
     OK = 'OK',
 }
+
+/**
+ * Enumeration defining the icon in a file badge.
+ */
+export enum FileBadgeType {
+    FEEDBACK_SUGGESTION = 'FEEDBACK_SUGGESTION',
```

Comment:
Why do we have an enum with just a single value?

Reply:
I added this enum to make it really easy later to other types of badges to the assessment code editor file tree. This way, we will be able to easily extend the badges introduced in this PR to also show badges on files that generally include any feedback, which I believe would be a nice addition in the future.